packages/pcd/zk-eddsa-frog-pcd/test/ZKEdDSAFrogPCD.spec.ts

```diff
@@ -309,11 +309,11 @@ describe("ZKEdDSAFrogPCD should work", function () {
     const validPCD = await ZKEdDSAFrogPCDPackage.prove(pcdArgs);
 
     await testVerifyBadClaim(validPCD, (claim: ZKEdDSAFrogPCDClaim) => {
-      claim.signerPublicKey[0] = "123";
+      claim.signerPublicKey = ["123", "456"];
     });
 
     await testVerifyBadClaim(validPCD, (claim: ZKEdDSAFrogPCDClaim) => {
-      claim.signerPublicKey[1] = "123";
+      claim.signerPublicKey = "123456";
```

Comment:
As elsewhere, I think this should be a well-formed signing key, or else it's not really testing the same thing.